pt.html

```diff
@@ -28,32 +28,32 @@
 - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
 
 <!DOCTYPE html>
-<html lang="en"> <!-- TRANSLATOR NOTE: Remember your 2-letter code? Replace "en" with that! -->
+<html lang="pt"> <!-- TRANSLATOR NOTE: Remember your 2-letter code? Replace "en" with that! -->
 <head>
 
 	<!-- Meta Stuff -->
-	<title>How To Remember Anything Forever-ish</title>
+	<title>Como Se Lembrar De Qualquer coisa Para Quase Sempre</title>
```

Comment:
```suggestion
	<title>Como Se Lembrar De Qualquer Coisa Para (Quase) Sempre</title>
```

Acredito que `Coisa` deveria estar com maiúsculo também e acho que o quase entre parenteses faz mais sentido para passar a mesma ideia do `Forever-ish`

Reply:
Sim, vou por o "Coisa" em maiúsculo. Porém o título em si ainda está em aberto (ou seja, da ate pra pensar algo diferente além de por coisa como fiz).

Até no repositório original você vai ver eu perguntando o que as outras pessoas estão usando para traduzir.